assignments/objects.js

```diff
@@ -19,27 +20,61 @@ const example = {
 
 // Write your intern objects here:
 
+const Intern = function(id, email, first_name, gender){
+  this.id = id,
+  this.email = email,
+  this.first_name = first_name,
+  this.gender = gender
+};
+
+const interns = [
+new Intern(1, "mmelloy0@psu.edu", "Mitzi", "F"),
+new Intern(2, "kdiben1@tinypic.com", "Kennan", "M"),
+new Intern(3, "kmummer2@wikimedia.org", "Keven", "M"),
+new Intern(4, "gmartinson3@illinois.edu", "Gannie", "M"),
+new Intern(5,"adaine5@samsung.com","Antonietta","F")
+];
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+let intern = interns.find(intern => intern.first_name === "Mitzi");
+console.log(intern.first_name);
 
 // Kennan's ID
+intern = interns.find(intern => intern.first_name === "Kennan");
+console.log(intern.id);
 
 // Keven's email
+intern = interns.find(intern => intern.first_name === "Keven");
+console.log(intern.email);
 
 // Gannie's name
+intern = interns.find(intern => intern.first_name === "Gannie");
+console.log(intern.first_name);
 
 // Antonietta's Gender
+intern = interns.find(intern => intern.first_name === "Antonietta");
+console.log(intern.gender);
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
+intern = interns.find(intern => intern.first_name === "Kennan");
+intern.speak = function(){return `Hello, my name is ${this.first_name}`};
```

Comment:
after reviewing your code I now understand what you were trying to do, I was a bit confused what you were referring to during stand up. so in order for `speak` method to be inherited from the Intern constructor. you would need to do use the prototype property. this is who it would look like

```Intern.prototype.speak = function(){return `Hello, my name is ${this.first_name}`};```
and now all the objects created from this Intern constructor can now use the `speak` method.